internal/store/node.go

```diff
@@ -59,6 +60,7 @@ var (
 								n.Status.NodeInfo.KubeletVersion,
 								n.Status.NodeInfo.KubeProxyVersion,
 								n.Spec.ProviderID,
+								n.Spec.PodCIDR,
```

Comment:
This is an optional label, not sure how we usually handle optional labels cc @brancz @tariq1890

Reply:
It’s not a pointer so at worst this is the empty string in which case Prometheus just drops the label so it’s fine as is.